tests/conftest.py

```
@@ -89,66 +114,121 @@ def _get_jwt_auth_headers(server_url: str, user_name: str | None = None) -> dict
+
+    jwt_token = auth_headers["Authorization"].replace("Bearer ", "")
+
+    sio = socketio.Client()
+    sio.connect(server, auth={"token": jwt_token}, wait=True)
+
+    try:
+        response = sio.call("room:join", {"roomId": room, "clientType": "frontend"})
+
+        # Handle room not found - create it first
+        if response.get("code") == 404:
+            create_response = requests.post(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)